openquake/calculators/postproc/compute_rtgm.py

```diff
@@ -326,6 +340,284 @@ def get_asce41(dstore, mce, facts):
             'BSE1N_S1': BSE1N_S1}
 
 
+def _get_label(imt):
+    imtlab = imt if imt == 'PGA' else imt.replace(')', 's)')
+    comp = 'Geom. mean' if imt == 'PGA' else 'Max. comp.'
+    return imtlab + ' - ' + comp
+
+
+def plot_meanHCs_afe_RTGM(imls, AFE, UHGM_RP, afe_RP, RTGM, afe_RTGM,
+                          imt_list):
+    plt.figure(figsize=(12, 9))
+    plt.rcParams.update({'font.size': 16})
+    colors = mpl.colormaps['viridis'].reversed()._resample(3)
+    patterns = ['-', '--', ':']
+    for i, imt in enumerate(imt_list):
+        lab = _get_label(imt)
+        plt.loglog(imls[i], AFE[i], color=colors(i), linestyle=patterns[i],
+                   label=lab, linewidth=3, zorder=1)
+        if i == 2:
```

Comment:
This is not readable at all. We should compare with the IMT corresponding to `i==2`